unit_test/fuzzing/test_secured_message/test_spdm_encode_secured_message/spdm_encode_secured_message.c

```diff
@@ -93,11 +94,6 @@ void libspdm_run_test_harness(void *test_buffer, size_t test_buffer_size)
         /* buffer too small */
         return;
     }
-    if (buffer_size >
-        LIBSPDM_MAX_MESSAGE_BUFFER_SIZE - record_header_max_size - aead_tag_max_size) {
-        buffer_size = LIBSPDM_MAX_MESSAGE_BUFFER_SIZE - record_header_max_size -
-                      aead_tag_max_size;
-    }
 
```

Comment:
why remove this check?

Reply:
test_buffer is the AUTH session, this check puts test_buffer as AppData
https://github.com/DMTF/libspdm/blob/bd5cde40aad7df7d07a54df5b11664d0fb53b1ed/include/industry_standard/spdm_secured_message.h#L37-L52

Reply:
If APP data is not consumed, why you care?
Basically, you just fuzz the App_Length.